src/handlers/notification/index.js

```diff
@@ -23,16 +23,21 @@
 'use strict'
 const Consumer = require('@mojaloop/central-services-shared').Kafka.Consumer
 const Logger = require('@mojaloop/central-services-shared').Logger
-const Config = require('../../lib/config')
+const Helper = require('../../lib/helper')
```

Comment:
I believe that we should follow the same approach that we access the DB. Please create DAO for this purpose. Internally it will call the caching services.

Let me know what you think about this approach.